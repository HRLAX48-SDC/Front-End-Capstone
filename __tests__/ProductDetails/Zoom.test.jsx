import React from 'react';
import renderer from 'react-test-renderer';
import Zoom from '../../client/src/components/productdetails/Zoom.jsx';

test('renders Zoom component correctly', () => {
  const zoom = renderer.create(<Zoom
    img={`images/placeholder-image.png`}
    zoomScale={2.5}
    height={650}
    width={900}
    transitionTime={0.5}
  />).toJSON();
  expect(zoom).toMatchSnapshot();
});