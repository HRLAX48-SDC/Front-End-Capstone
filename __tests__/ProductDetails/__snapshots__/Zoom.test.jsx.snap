// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders Zoom component correctly 1`] = `
<div
  className="p_pic_expanded_zoom"
  onMouseMove={[Function]}
  onMouseOut={[Function]}
  onMouseOver={[Function]}
  style={
    Object {
      "height": "650px",
      "overflow": "hidden",
      "width": "900px",
    }
  }
>
  <div
    style={
      Object {
        "backgroundImage": "url('images/placeholder-image.png')",
        "backgroundPosition": "center",
        "backgroundRepeat": "no-repeat",
        "backgroundSize": "auto 100%",
        "height": "650px",
        "transform": "scale(1.0)",
        "transformOrigin": "null% null%",
        "transition": "transform 0.5s ease-out",
      }
    }
  />
</div>
`;
